api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -40,6 +40,15 @@ message TCPAccessLogEntry {
 
   // Properties of the TCP connection.
   ConnectionProperties connection_properties = 2;
+
+  // If this log entry is final log entry that flushed after the connection closed or
+  // intermediate log entry that flushed periodically during the connection.
+  // There may be multiple intermediate log entries and only one final log entry for each
+  // TCP connection.
+  // And if it is necessary, some IDs or identifiers can be added to the log entry
+  // :ref:`metadata <envoy_v3_api_field_data.accesslog.v3.AccessLogCommon.metadata>` to
+  // correlate all these intermediate log entries and final log entry.
+  bool intermediate_log_entry = 3;
```

Comment:
Another thought, this might be in `common_properties`, IIRC there are similar request for long-live H2 stream (gRPC stream).

Reply:
ha, yeah, I will move it to the common_properties.